datasync/chaindatafetcher/kafka/kafka_test.go

```diff
@@ -40,7 +40,7 @@ func (s *KafkaSuite) SetupTest() {
 	config.Brokers = []string{"kafka:9094"}
 	kfk, err := NewKafka(config)
 	if err == sarama.ErrOutOfBrokers {
-		s.T().Log("Failed connecting to brokers", config.brokers)
+		s.T().Log("Failed connecting to brokers", config.Brokers)
```

Comment:
I merged the PR https://github.com/klaytn/klaytn/pull/695. Please rebase it.

Reply:
Thanks for your comment. I rebased it.